readme.txt

```diff
@@ -22,11 +22,6 @@ This section describes how to install the plugin and get it working.
 
 == Changelog ==
 
-= Unreleased =
-* Use the theme color for completed task strikethrough #1006
-* Add free trial notice on WooCommerce Orders page #936
-* Change illustration in tasklist completed component #1007
-
```

Comment:
Hey @ilyasfoo Don't we need to add changelog? Or we just add changelogs in release PR?

Reply:
@chihsuan Doing this due to the coordination to release 2.0.5 in  p1677667881815459/1677664953.596819-slack-C03Q87XT1QF

Reply:
Not the most ideal process tbh, but we haven't really discussed on how to improve it yet

Reply:
Ah, Gotcha, thanks!